docs/BATCHING.md

```diff
@@ -66,11 +56,10 @@ In the undesirable cases where Solr becomes unresponsive (e.g., data corruption,
 
 If a Solr core has become unresponsive and the specified error threshold has been traversed, and if, in addition, the high water mark has been exceeded, then the yokozuna batching system has a mechanism for automatically purging enqueued entries, so as to allow vnodes to continue servicing requests, as well as to allow update operations to occur for indices that are not in a pathological state.
 
-The yokozuna batching subsystem supports 4 different purge strategies:
+The yokozuna batching subsystem supports 3 different purge strategies:
 
 * `purge_one` (default behavior): Purge the oldest entry from a randomly selected indexq structure among the set of search indexes which have crossed the error threshold;
```

Comment:
This no longer is "a randomly selected indexq structure" as there are A) no indexq structures and B) it'll always purge from the {index, partition} to which the new entry is written.